src/App.css

```diff
@@ -1,3 +1,197 @@
+
+.App {
+  background-color: #f5f6fa;
+  padding: 9.313rem 6.25rem;
+}
+
+.box {
+  background-color: #fafafa;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+}
+
+.header-text {
+  font-family: 'Montserrat', sans-serif;
+  color: #0290ff;
+  font-size: 1.875rem;
+  margin: 0;
+  padding-right: 2.563rem;
+}
+
+.link {
+  text-decoration: none;
+  color: gray;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.813rem;
+  font-weight: normal;
+  font-stretch: normal;
+  letter-spacing: 1.9px;
+  margin: 0 2.563rem 0 0;
+}
+
+.link:hover {
+  color: black;
+}
+
+.book-panel {
+  width: 75rem;
+  margin: auto;
+  margin-top: 1.25rem;
+  box-shadow: 0 0 60px 0 rgba(0, 0, 0, 0.05);
+  border: 1px solid #e8e8e8;
+  padding: 1.5rem;
+}
+
+.book-genre {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+}
+
+.book-title {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 0 0 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+}
+
+.book-author {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.book-link {
+  text-decoration: none !important;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.progress-oval {
+  position: relative;
+  border: 4px solid;
+  width: 4.25rem;
+  height: 4.25rem;
+  border-right-color: #307bbe;
+  border-left-color: white;
+  border-top-color: #307bbe;
+  border-bottom-color: white;
+  transform: rotate(30deg);
+  border-radius: 50%;
+}
+
+.percent-complete {
+  width: 4.25rem;
+  height: 2.438rem;
+  margin: 0.063rem 5.75rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: lighter;
+}
+
+.completed {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0.75rem 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+}
+
+.current-chapter {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+}
+
+.current-lesson {
+  width: 4.688rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: 'RobotoSlab', serif;
+  font-size: 1rem;
+  font-weight: 300;
+  letter-spacing: -0.4px;
+}
+
+.btn-update {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 0 0.563rem 3.688rem;
+  padding: 0.3rem 0.7rem 0.5rem 1rem !important;
+  border-radius: 3px;
+  background-color: #0290ff !important;
+  font-family: 'RobotoSlab', serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff !important;
```

Comment:
- [ ] Please, don't use ! important tag, this is a bad practice and change the CSS flow on the project. 🌮